python/sglang/srt/entrypoints/engine.py

```diff
@@ -283,9 +283,13 @@ def flush_cache(self):
         loop = asyncio.get_event_loop()
         return loop.run_until_complete(self.tokenizer_manager.flush_cache())
 
-    def start_profile(self):
+    def start_profile(self, activities: Optional[List[str]] = None):
+        if activities is None:
+            activities = ["CPU", "GPU"]
```

Comment:
why?

Reply:
revert the changes and leave it for future pr